api/v1/config.js

```diff
@@ -171,7 +171,7 @@ module.exports = {
   },
   links: [
     {
-      url: 'https://discord.gg/MWtDws8',
+      url: 'https://discord.gg/N49F7wJ', // Taken from #information Discord channel on 6/5/2020.
```

Comment:
@InoUno @danielvarcas What is the ideal way to centralize this hyperlink?

Reply:
The url is defined in two places. He's asking where we'd prefer to put it so that if it needs updating it only needs to be updated in once place.

Reply:
@bkesecker @slassen Ah sorry. So you could define it as a variable in the same file above the module.exports block. i.e. `const discordUrl = 'https://discord.gg/N49F7wJ'`. And then reference the variable instead, i.e. `discord: discordUrl`.

Reply:
The perfectionist in me who always defines global constants as such.  Not going to sweat it.